src/cai_lab_to_nwb/zaki_2024/zaki_2024_notes.md

```diff
@@ -776,6 +776,8 @@ This is the metadata file in the same directory that the minian videos:
 }
 ```
 
+Something concerning is that the 30FPS does not correspond to the videos. The metadata says 30 FPS but the videos are 60 FPS.
+
```

Comment:
I need a clarification: The calcium imaging videos are 60 fps and the behavioral videos are 30 fps, or is it the opposite?

Reply:
I think I checked against OpenCV, why are you suspicious?

Reply:
(looking at subjkect EEG3-4) Because 60fps is a bit unusual for onephoton imaging (higher that the usual 20-30 Hz) + the timestamps are 33 ms apart (even if they are not always regular)  that is 30Hz + if you account for 30fps, the time span of the calcium imaging recording is the same of the behavioral recording which would make more sense instead of recording with miniscope just half of the time (as it would be accounting for 60fps)

Reply:
Thanks for writing this. Let me take a close look.